argusdashboard/src/AppBundle/Controller/Configuration/GenerateController.php

```diff
@@ -108,6 +109,8 @@ private function handleAlertForm($data)
                 $diseaseValue = $diseaseValueService->getById($match[2]);
                 if (!$diseaseValue) // disease value not found
                 {
+                    $logger = $this->get('logger');

```

Comment:
If the function getLogger doens't exist in the BaseController , could you please create it ? and then use it 
```
public function getLogger()
    {
        return $this->get('logger');
    }
```